python_modules/dagster/dagster/_generate/templates/CODE_LOCATION_NAME_PLACEHOLDER/pyproject.toml.tmpl

```diff
@@ -1,7 +1,25 @@
+[project]
+name = "{{ code_location_name }}"
+version = "0.1.0"
+description = "Add your description here"
+readme = "README.md"
+requires-python = ">=3.8,<3.12"
+dependencies = [
+    "dagster",
+    "dagster-cloud",
```

Comment:
should this be optional, like behind a flag?

Reply:
Ideally the dependencies would match those in the setup.py file. Are they defined as a variable in scaffold and the examples?

Reply:
Should they match this file?
https://github.com/dagster-io/dagster/blob/master/python_modules/dagster/dagster/_generate/templates/REPO_NAME_PLACEHOLDER/setup.py.tmpl